pinot-minion/src/main/java/org/apache/pinot/minion/MinionStarter.java

```diff
@@ -68,6 +71,8 @@
   private final HelixManager _helixManager;
   private final TaskExecutorFactoryRegistry _taskExecutorFactoryRegistry;
   private final EventObserverFactoryRegistry _eventObserverFactoryRegistry;
+  private MinionAdminApiApplication _minionAdminApplication;
+  private final List<ListenerConfig> _listenerConfigs;
 
   public MinionStarter(String helixClusterName, String zkAddress, PinotConfiguration config)
       throws Exception {
```

Comment:
You'll need one additional call to `installDefaultSSLFactory` here. see ControllerStarter `setUpPinotController()`